src/components/button.tsx

```diff
@@ -2,9 +2,14 @@ import React, { forwardRef, type ElementRef, type ComponentProps } from "react";
 
 const defaultTag = "button";
 
+export type ButtonProps = ComponentProps<typeof defaultTag>
+
 export const Button = forwardRef<
   ElementRef<typeof defaultTag>,
-  ComponentProps<typeof defaultTag>
->((props, ref) => <button {...props} ref={ref} />);
+    ButtonProps
+>((props, ref) => <button style={{border:"none"}} {...props} ref={ref} />);
 
+Button.defaultProps = {
+    type: "button", // otherwise it will be submit
```

Comment:
interesting, should we keep it submit since its the browser default?

Reply:
Normally I’d say no but if we thinking about defaults then it’s more aligned yeah